firefox-ios/Client/Frontend/Settings/AppearanceSettings/AppearanceSettingsView.swift

```diff
@@ -169,4 +215,15 @@ struct AppearanceSettingsView: View, FeatureFlaggable {
             // TODO(FXIOS-11584): Add telemetry here
         }
     }
+
+    /// Updates the middle button in navigation toolbar based on the user's selection.
+    /// - Parameter selectedOption: The selected theme option from ThemeSelectionView.
+    private func updateMiddleNavigationToolbarButton(to selectedOption: NavigationBarMiddleButtonType) {
+        prefs.setString(selectedOption.rawValue, forKey: AppConstants.prefNavigationToolbarMiddleButton)
```

Comment:
Curious, why `AppConstants` vs. `PrefsKeys`?

Reply:
Good point. Move it to PrefKeys - it fits better there.